applications/emocam/index.html

```diff
@@ -711,29 +713,99 @@
     } else {
         console.warn("getUserMedia() is not supported by your browser");
     }
-
+   $(document).on("click", ".vid_link", function (event) {
+        var videoElement = document.querySelector('video');
+        hrefValue = $(this).text();
+        var idValue = $(this).attr('id');
+        vidopttrigger = true;
+        document.querySelector("#videoSource").value=idValue;
+        $("#videoSource").val(idValue).prop('selected', true);
+        $("#videoSource").val(idValue).change();
+        $("#videoDropdown").val(hrefValue);
+        MyAvatar.removeAnimationStateHandler(handlerId);
+        MyAvatar.restoreAnimation();
```

Comment:
Why are there MyAvatar calls inside the QtWebengine part? The API is available only on the script engine part of the application so this will fail.

Reply:
Thanks for catching that... I have no idea what that was supposed to do or how it got in there.